src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -258,6 +260,15 @@ const UniqueTokenExpandedState = ({
       });
   }, [asset.network, isPoap, network, urlSuffixForAsset]);
 
+  useEffect(() => {
```

Comment:
After the ENS expanded state PR got merged, I wonder if you could just do:

```tsx
const isOwner = useMemo(() => ensProfile.data?.owner?.address === accountAddress, []);
```

Reply:
added!